### ISSUE TITLE
Scroll with webpage

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#232]
```
What steps will reproduce the problem?
1.Focus on flexpaper player
2.Scroll mouse wheel
3.Both player and webpage scroll 

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
Firefox 8 and Chrome 15
Windows 7

Please provide any additional information below.

```

Original issue reported on code.google.com by `liy...@gmail.com` on 21 Nov 2011 at 11:46

**Comment by GoogleCodeExporter** [comment]
```
Cannot reproduce this, can you reproduce this on our start page ? 
(http://flexpaper.devaldi.com)
```

Original comment by `erik.eng...@devaldi.com` on 24 Nov 2011 at 12:38

**Comment by GoogleCodeExporter** [comment]
```
It works fine with http://flexpaper.devaldi.com/

But here I set up a live demo with this issue. I use FlexPaper_1.4.7_flash.zip 
and change one <div>'s height, the rest parts are completely untouched.

https://flexpaper-test.googlecode.com/svn/trunk/index.html
```

Original comment by `liy...@gmail.com` on 24 Nov 2011 at 5:42

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@devaldi.com` on 29 Dec 2011 at 5:23
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
I have the same happen (I think?)

Flexpaper doesn't take all the focus - I would like to be able to click or 
hover over flexpaper, and using the scroll wheel, scroll the flexpaper 
document. As it is at the moment, you have to hit to the bottom of the page 
before flexviewer will start scrolling.

Thanks
John
```

Original comment by `johnwise...@gmail.com` on 30 Dec 2011 at 10:17

**Comment by GoogleCodeExporter** [comment]
```
Hey guys, I found a project related to this issue, maybe it could help.

http://code.google.com/p/mousewheeltrap/
```

Original comment by `liy...@gmail.com` on 8 Mar 2012 at 9:08

**Comment by GoogleCodeExporter** [comment]
```
And FlexPaper Zine has the scrolling issue too.
```

Original comment by `liy...@gmail.com` on 7 Apr 2012 at 3:59

**Comment by GoogleCodeExporter** [comment]
```
fixed and will be available in upcoming build 1.5.2
```

Original comment by `erik.eng...@devaldi.com` on 7 Jun 2012 at 11:13
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
It still doesn't work in 1.5.8. I tested it in IE 9. It works in all other 
browsers. I haven't tested older versions of IE though.
```

Original comment by `bhamade-...@simplifymd.com` on 5 Oct 2012 at 2:20
